numpy requires Python '>=3.8' but the running Python is 3.6.8
[I 18:44:22.883 NotebookApp] Serving notebooks from local directory: /dcs/large/u2145461/cs407/Machine-Unlearning-x-Continual-Learning
[I 18:44:22.884 NotebookApp] Jupyter Notebook 6.5.2 is running at:
[I 18:44:22.884 NotebookApp] http://gecko-01:11861/
[I 18:44:22.885 NotebookApp] Use Control-C to stop this server and shut down all kernels (twice to skip confirmation).
[I 18:45:35.894 NotebookApp] 302 GET / (137.205.113.129) 3.460000ms
[I 18:45:36.212 NotebookApp] 302 GET /tree? (137.205.113.129) 4.160000ms
[I 18:45:43.345 NotebookApp] 302 POST /login? (137.205.113.129) 273.710000ms
[I 18:46:15.793 NotebookApp] Kernel started: 97d8d1d2-20e6-4259-9bad-3226a53045b4, name: python3
[I 18:46:23.669 NotebookApp] Starting buffering for 97d8d1d2-20e6-4259-9bad-3226a53045b4:3b8d681a-7360-4a70-a009-df9b755b3f27
[I 18:46:23.694 NotebookApp] Restoring connection for 97d8d1d2-20e6-4259-9bad-3226a53045b4:3b8d681a-7360-4a70-a009-df9b755b3f27
Traceback (most recent call last):
  File "/local/java/python-ml/22-12-21-python3.9/lib/python3.9/site-packages/IPython/core/completer.py", line 2807, in _complete
    result = cast(matcher, MatcherAPIv2)(context)
  File "/local/java/python-ml/22-12-21-python3.9/lib/python3.9/site-packages/IPython/core/completer.py", line 1762, in magic_matcher
    matches = self.magic_matches(text)
  File "/local/java/python-ml/22-12-21-python3.9/lib/python3.9/site-packages/IPython/core/completer.py", line 1795, in magic_matches
    global_matches = self.global_matches(bare_text)
  File "/local/java/python-ml/22-12-21-python3.9/lib/python3.9/site-packages/IPython/core/completer.py", line 1003, in global_matches
    for word in lst:
RuntimeError: dictionary changed size during iteration
[I 19:21:01.223 NotebookApp] Kernel restarted: 97d8d1d2-20e6-4259-9bad-3226a53045b4
[I 19:21:01.243 NotebookApp] Starting buffering for 97d8d1d2-20e6-4259-9bad-3226a53045b4:3b8d681a-7360-4a70-a009-df9b755b3f27
[I 19:21:01.266 NotebookApp] Restoring connection for 97d8d1d2-20e6-4259-9bad-3226a53045b4:3b8d681a-7360-4a70-a009-df9b755b3f27
[W 19:21:05.835 NotebookApp] Nudge: attempt 10 on kernel 97d8d1d2-20e6-4259-9bad-3226a53045b4
[I 19:45:58.637 NotebookApp] Kernel restarted: 97d8d1d2-20e6-4259-9bad-3226a53045b4
[I 19:45:58.659 NotebookApp] Starting buffering for 97d8d1d2-20e6-4259-9bad-3226a53045b4:3b8d681a-7360-4a70-a009-df9b755b3f27
[I 19:45:58.688 NotebookApp] Restoring connection for 97d8d1d2-20e6-4259-9bad-3226a53045b4:3b8d681a-7360-4a70-a009-df9b755b3f27
[W 19:46:03.234 NotebookApp] Nudge: attempt 10 on kernel 97d8d1d2-20e6-4259-9bad-3226a53045b4
[I 20:11:48.118 NotebookApp] Kernel restarted: 97d8d1d2-20e6-4259-9bad-3226a53045b4
[I 20:11:48.139 NotebookApp] Starting buffering for 97d8d1d2-20e6-4259-9bad-3226a53045b4:3b8d681a-7360-4a70-a009-df9b755b3f27
[I 20:11:48.162 NotebookApp] Restoring connection for 97d8d1d2-20e6-4259-9bad-3226a53045b4:3b8d681a-7360-4a70-a009-df9b755b3f27
[W 20:11:52.705 NotebookApp] Nudge: attempt 10 on kernel 97d8d1d2-20e6-4259-9bad-3226a53045b4
